src/controller/CHIPDeviceController.cpp

```diff
@@ -184,9 +184,15 @@ CHIP_ERROR DeviceController::Init(NodeId localDeviceId, ControllerInitParams par
     );
     SuccessOrExit(err);
 
+    err = mAdmins.Init(mStorageDelegate);
```

Comment:
`mAdmins` is really `mFabricContexts`. 

Code like:

```
admin = mAdmins.AssignAdminId(mAdminId, localDeviceId);
```
The `admin` object is not your administrator. It's more a fabric context.

```
admin->SetOperationalCert(ByteSpan(chipCert, chipCertLen)
```

would read better as:
```
fabric->SetOperationalCert(ByteSpan(chipCert, chipCertLen)
```

And would be much closer to spec. This is not necessarily for this PR, but overall, we should soon use terminology that is now accepted and balloted.

@bzbarsky-apple @andreilitvin

Reply:
I agree. I'll submit an issue so that we don't keep pushing it down the road.

Reply:
https://github.com/project-chip/connectedhomeip/issues/6826